xds/internal/xdsclient/rds_test.go

```diff
@@ -714,11 +719,11 @@ func (s) TestUnmarshalRouteConfig(t *testing.T) {
 					VirtualHosts: []*VirtualHost{
 						{
 							Domains: []string{uninterestingDomain},
-							Routes:  []*Route{{Prefix: newStringP(""), WeightedClusters: map[string]WeightedCluster{uninterestingClusterName: {Weight: 1}}}},
+							Routes:  []*Route{{Prefix: newStringP(""), WeightedClusters: map[string]WeightedCluster{uninterestingClusterName: {Weight: 1}}, RouteAction: RouteActionRoute}},
```

Comment:
Optional: Same comment as in `v2/rds_test.go`.

Reply:
Done.